src/main/java/net/apnic/rdap/authority/config/AuthorityConfiguration.java

```diff
@@ -221,20 +232,15 @@ private void setupAuthorityStore()
 
             if(aConfig.getServers() != null)
             {
-                List<URI> servers = aConfig.getServers().stream()
-                    .map((String strURI) ->
-                    {
-                        try
-                        {
-                            return new URI(strURI);
-                        }
-                        catch(URISyntaxException ex)
-                        {
-                            throw new RuntimeException(ex);
-                        }
-                    })
-                    .collect(Collectors.toList());
-                authority.addServers(servers);
+                authority.addServers(
+                    aConfig.getServers().stream()
+                    .map(URI::create)
+                    .collect(Collectors.toList()));
+            }
+
+            if(aConfig.getDefaultServer() != null)
+            {
+                authority.setDefaultServer(URI.create(aConfig.getDefaultServer()));
             }
 
             authorityStore().addAuthority(authority);
```

Comment:
Yes - In that you would have to make RDAPAuthority an immutable or at least disallow changes to an instance after it has been added to the store. Dur me. :p